Mathlib/Algebra/Algebra/NonUnitalSubalgebra.lean

```diff
@@ -312,12 +312,18 @@ def toSubmoduleEquiv (S : NonUnitalSubalgebra R A) : S.toSubmodule ≃ₗ[R] S :
 
 variable [FunLike F A B] [NonUnitalAlgHomClass F R A B]
 
+/-- The LinearMap underlying a NonUnitalAlgHom -/
+def _root_.NonUnitalAlgHomClass.toLinearMap (f : F) : A →ₗ[R] B where
+  toFun := f
+  map_add' := map_add f
+  map_smul' := map_smulₛₗ f
```

Comment:
Why not, yes.

Reply:
In fact, `exact?` finds `exact LinearMapClass.linearMap f`
so I wonder whether we have to add it explicitly. What do you think?

Reply:
And since the dot notation does not work and it is shorter to write `LinearMapClass.linearMap f` than `NonUnialAlgHomClass.toLinearMap f`, I delete it.